### PR TITLE
Feature/issue 49 fix test cases

### LINKED ISSUE #49: fixing current test case
### Issue description
Tell us what should happen and what happens instead

### Example code
```go
If possible, please enter some example code here to reproduce the issue.
```

### Error log
```
If you have an error log, please paste it here.
```

### Configuration
*OS:*

*Interface:* E.g. `native`, `database/sql`

*Driver version:*

*Go version:* run `go version` in your console

*ClickHouse Server version:*

### PR DESCRIPTION
**Pull request by leo-cai-timeplus**
* alter port
* alter query sqls
* fix date type

### CODE REVIEW

File: tests/bigint_test.go [hunk 1]
```diff
@@ -124,10 +124,10 @@ func TestNullableBigInt(t *testing.T) {
 		})
 	)
 	if assert.NoError(t, err) {
-		if err := checkMinServerVersion(conn, 1, 0); err != nil {
-			t.Skip(err.Error())
-			return
-		}
+		//if err := checkMinServerVersion(conn, 1, 0); err != nil {
```
Comment (qijun-niu-timeplus) [inline tests/bigint_test.go]: delete this block

File: tests/columnar_batch_test.go [hunk 2]
```diff
@@ -114,10 +114,11 @@ func TestColumnarInterface(t *testing.T) {
 }
 
 func TestNullableColumnarInterface(t *testing.T) {
+	t.Skip("TRUNCATE unable to delete data in logstore")
```
Comment (qijun-niu-timeplus) [inline tests/columnar_batch_test.go]: Proton doesn't support TRUNCATE operation for streaming query

File: tests/geo_multipolygon_test.go [hunk 3]
```diff
@@ -45,21 +46,21 @@ func TestGeoMultiPolygon(t *testing.T) {
 		})
 	)
 	if assert.NoError(t, err) {
-		if err := checkMinServerVersion(conn, 99, 99); err != nil {
-			t.Skip(err.Error())
-			return
-		}
+		//if err := checkMinServerVersion(conn, 99, 99); err != nil {
```
Comment (qijun-niu-timeplus) [inline tests/geo_multipolygon_test.go]: remove

File: tests/geo_point_test.go [hunk 4]
```diff
@@ -45,21 +46,21 @@ func TestGeoPoint(t *testing.T) {
 		})
 	)
 	if assert.NoError(t, err) {
-		if err := checkMinServerVersion(conn, 99, 99); err != nil {
-			t.Skip(err.Error())
-			return
-		}
+		//if err := checkMinServerVersion(conn, 99, 99); err != nil {
```
Comment (qijun-niu-timeplus) [inline tests/geo_point_test.go]: remove

File: tests/geo_polygon_test.go [hunk 5]
```diff
@@ -45,21 +46,21 @@ func TestGeoPolygon(t *testing.T) {
 		})
 	)
 	if assert.NoError(t, err) {
-		if err := checkMinServerVersion(conn, 99, 99); err != nil {
-			t.Skip(err.Error())
-			return
-		}
+		//if err := checkMinServerVersion(conn, 99, 99); err != nil {
```
Comment (qijun-niu-timeplus) [inline tests/geo_polygon_test.go]: remove

File: tests/geo_ring_test.go [hunk 6]
```diff
@@ -45,21 +46,21 @@ func TestGeoRing(t *testing.T) {
 		})
 	)
 	if assert.NoError(t, err) {
-		if err := checkMinServerVersion(conn, 99, 99); err != nil {
-			t.Skip(err.Error())
-			return
-		}
+		//if err := checkMinServerVersion(conn, 99, 99); err != nil {
```
Comment (qijun-niu-timeplus) [inline tests/geo_ring_test.go]: remove

File: tests/issues/389_test.go [hunk 7]
```diff
@@ -43,25 +43,25 @@ func TestIssue389(t *testing.T) {
 		})
 	)
 	if assert.NoError(t, err) {
-		if err := checkMinServerVersion(conn, 20, 3); err != nil {
-			t.Skip(err.Error())
-			return
-		}
+		//if err := checkMinServerVersion(conn, 20, 3); err != nil {
```
Comment (qijun-niu-timeplus) [inline tests/issues/389_test.go]: remove comments

File: tests/issues/406_test.go [hunk 8]
```diff
@@ -42,20 +42,20 @@ func TestIssue406(t *testing.T) {
 		})
 	)
 	if assert.NoError(t, err) {
-		if err := checkMinServerVersion(conn, 21, 9); err != nil {
-			t.Skip(err.Error())
-			return
-		}
+		//if err := checkMinServerVersion(conn, 21, 9); err != nil {
```
Comment (qijun-niu-timeplus) [inline tests/issues/406_test.go]: remove comments

File: tests/issues/412_test.go [hunk 9]
```diff
@@ -43,27 +43,27 @@ func TestIssue412(t *testing.T) {
 		})
 	)
 	if assert.NoError(t, err) {
-		if err := checkMinServerVersion(conn, 21, 9); err != nil {
-			t.Skip(err.Error())
-			return
-		}
+		//if err := checkMinServerVersion(conn, 21, 9); err != nil {
```
Comment (qijun-niu-timeplus) [inline tests/issues/412_test.go]: remove comments

File: tests/issues/482_test.go [hunk 10]
```diff
@@ -66,10 +66,11 @@ func TestIssue482(t *testing.T) {
 			&int32OverDate,
 		)
 		if assert.NoError(t, err) {
-			assert.Equal(t, "2061-02-01 00:00:00", myIssueDate.Format("2006-01-02 15:04:05"))
-			assert.Equal(t, "2064-01-01 00:00:00", myIssueDateTo64.Format("2006-01-02 15:04:05"))
-			assert.Equal(t, "2038-01-19 05:14:07", int32MaxDate.Format("2006-01-02 15:04:05"))
-			assert.Equal(t, "2038-01-19 05:14:08", int32OverDate.Format("2006-01-02 15:04:05"))
+			assert.Equal(t, "2061-02-01 00:00:00", myIssueDate.UTC().Format("2006-01-02 15:04:05"))
```
Comment (qijun-niu-timeplus) [inline tests/issues/482_test.go:69]: Why do we have to change to UTC time zone?

File: tests/issues/485/main.go [hunk 11]
```diff
@@ -6,7 +6,7 @@
 // not use this file except in compliance with the License.
 // You may obtain a copy of the License at
 //
-//     http://www.apache.org/licenses/LICENSE-2.0
```
Comment (qijun-niu-timeplus) [inline tests/issues/485/main.go:9]: revert this line

File: tests/json_test.go [hunk 12]
```diff
@@ -97,20 +102,20 @@ func TestNullableJson(t *testing.T) {
 		})
 	)
 	if assert.NoError(t, err) {
-		if err := checkMinServerVersion(conn, 1, 0); err != nil {
-			t.Skip(err.Error())
-			return
-		}
+		//if err := checkMinServerVersion(conn, 99, 99); err != nil {
```
Comment (qijun-niu-timeplus) [inline tests/json_test.go]: remove comments

File: tests/simple_aggregate_function_test.go [hunk 13]
```diff
@@ -42,22 +42,22 @@ func TestSimpleAggregateFunction(t *testing.T) {
 		})
 	)
 	if assert.NoError(t, err) {
-		if err := checkMinServerVersion(conn, 99, 99); err != nil {
-			t.Skip(err.Error())
-			return
-		}
+		//if err := checkMinServerVersion(conn, 99, 99); err != nil {
```
Comment (qijun-niu-timeplus) [inline tests/simple_aggregate_function_test.go]: remove comments

File: tests/std/bigint_test.go [hunk 14]
```diff
@@ -26,11 +26,11 @@ import (
 )
 
 func TestStdBigInt(t *testing.T) {
-	if conn, err := sql.Open("proton", "proton://127.0.0.1:9000"); assert.NoError(t, err) {
-		if err := checkMinServerVersion(conn, 21, 12); err != nil {
-			t.Skip(err.Error())
-			return
-		}
+	if conn, err := sql.Open("proton", "proton://127.0.0.1:8463"); assert.NoError(t, err) {
+		//if err := checkMinServerVersion(conn, 21, 12); err != nil {
```
Comment (qijun-niu-timeplus) [inline tests/std/bigint_test.go]: remove comments

File: tests/std/bigint_test.go [hunk 15]
```diff
@@ -96,11 +96,11 @@ func TestStdBigInt(t *testing.T) {
 }
 
 func TestStdNullableBigInt(t *testing.T) {
-	if conn, err := sql.Open("proton", "proton://127.0.0.1:9000"); assert.NoError(t, err) {
-		if err := checkMinServerVersion(conn, 21, 12); err != nil {
-			t.Skip(err.Error())
-			return
-		}
+	if conn, err := sql.Open("proton", "proton://127.0.0.1:8463"); assert.NoError(t, err) {
+		//if err := checkMinServerVersion(conn, 21, 12); err != nil {
```
Comment (qijun-niu-timeplus) [inline tests/std/bigint_test.go]: remove comments

File: tests/std/bool_test.go [hunk 16]
```diff
@@ -25,19 +25,19 @@ import (
 )
 
 func TestStdBool(t *testing.T) {
-	if conn, err := sql.Open("proton", "proton://127.0.0.1:9000"); assert.NoError(t, err) {
-		if err := checkMinServerVersion(conn, 21, 12); err != nil {
-			t.Skip(err.Error())
-			return
-		}
+	if conn, err := sql.Open("proton", "proton://127.0.0.1:8463"); assert.NoError(t, err) {
+		//if err := checkMinServerVersion(conn, 21, 12); err != nil {
```
Comment (qijun-niu-timeplus) [inline tests/std/bool_test.go]: remove comments

File: tests/std/date32_test.go [hunk 17]
```diff
@@ -19,59 +19,62 @@ package std
...
 )
 
 func TestStdDate32(t *testing.T) {
-	if conn, err := sql.Open("proton", "proton://127.0.0.1:9000"); assert.NoError(t, err) {
-		if err := checkMinServerVersion(conn, 21, 9); err != nil {
-			t.Skip(err.Error())
-			return
-		}
+	if conn, err := sql.Open("proton", "proton://127.0.0.1:8463"); assert.NoError(t, err) {
+		//if err := checkMinServerVersion(conn, 21, 9); err != nil {
```
Comment (qijun-niu-timeplus) [inline tests/std/date32_test.go]: remove

File: tests/std/datetime64_test.go [hunk 18]
```diff
@@ -26,11 +26,11 @@ import (
 )
 
 func TestStdDateTime64(t *testing.T) {
-	if conn, err := sql.Open("proton", "proton://127.0.0.1:9000"); assert.NoError(t, err) {
-		if err := checkMinServerVersion(conn, 20, 3); err != nil {
-			t.Skip(err.Error())
-			return
-		}
+	if conn, err := sql.Open("proton", "proton://127.0.0.1:8463"); assert.NoError(t, err) {
+		//if err := checkMinServerVersion(conn, 20, 3); err != nil {
```
Comment (qijun-niu-timeplus) [inline tests/std/datetime64_test.go]: remove

File: tests/std/decimal_test.go [hunk 19]
```diff
@@ -26,18 +26,18 @@ import (
 )
 
 func TestStdDecimal(t *testing.T) {
-	if conn, err := sql.Open("proton", "proton://127.0.0.1:9000"); assert.NoError(t, err) {
-		if err := checkMinServerVersion(conn, 21, 1); err != nil {
-			t.Skip(err.Error())
-			return
-		}
+	if conn, err := sql.Open("proton", "proton://127.0.0.1:8463"); assert.NoError(t, err) {
+		//if err := checkMinServerVersion(conn, 21, 1); err != nil {
```
Comment (qijun-niu-timeplus) [inline tests/std/decimal_test.go]: remove

File: tests/std/geo_multipolygon_test.go [hunk 20]
```diff
@@ -28,19 +28,20 @@ import (
...
 	ctx := proton.Context(context.Background(), proton.WithSettings(proton.Settings{
 		"allow_experimental_geo_types": 1,
 	}))
-	if conn, err := sql.Open("proton", "proton://127.0.0.1:9000"); assert.NoError(t, err) {
-		if err := checkMinServerVersion(conn, 21, 12); err != nil {
-			t.Skip(err.Error())
-			return
-		}
+	if conn, err := sql.Open("proton", "proton://127.0.0.1:8463"); assert.NoError(t, err) {
+		//if err := checkMinServerVersion(conn, 21, 12); err != nil {
```
Comment (qijun-niu-timeplus) [inline tests/std/geo_multipolygon_test.go]: remove

File: tests/std/geo_point_test.go [hunk 21]
```diff
@@ -28,19 +28,20 @@ import (
...
 	ctx := proton.Context(context.Background(), proton.WithSettings(proton.Settings{
 		"allow_experimental_geo_types": 1,
 	}))
-	if conn, err := sql.Open("proton", "proton://127.0.0.1:9000"); assert.NoError(t, err) {
-		if err := checkMinServerVersion(conn, 21, 12); err != nil {
-			t.Skip(err.Error())
-			return
-		}
+	if conn, err := sql.Open("proton", "proton://127.0.0.1:8463"); assert.NoError(t, err) {
+		//if err := checkMinServerVersion(conn, 21, 12); err != nil {
```
Comment (qijun-niu-timeplus) [inline tests/std/geo_point_test.go]: remove

File: tests/std/geo_polygon_test.go [hunk 22]
```diff
@@ -28,19 +28,20 @@ import (
...
 	ctx := proton.Context(context.Background(), proton.WithSettings(proton.Settings{
 		"allow_experimental_geo_types": 1,
 	}))
-	if conn, err := sql.Open("proton", "proton://127.0.0.1:9000"); assert.NoError(t, err) {
-		if err := checkMinServerVersion(conn, 21, 12); err != nil {
-			t.Skip(err.Error())
-			return
-		}
+	if conn, err := sql.Open("proton", "proton://127.0.0.1:8463"); assert.NoError(t, err) {
+		//if err := checkMinServerVersion(conn, 21, 12); err != nil {
```
Comment (qijun-niu-timeplus) [inline tests/std/geo_polygon_test.go]: remove

File: tests/std/geo_ring_test.go [hunk 23]
```diff
@@ -28,19 +28,20 @@ import (
...
 	ctx := proton.Context(context.Background(), proton.WithSettings(proton.Settings{
 		"allow_experimental_geo_types": 1,
 	}))
-	if conn, err := sql.Open("proton", "proton://127.0.0.1:9000"); assert.NoError(t, err) {
-		if err := checkMinServerVersion(conn, 21, 12); err != nil {
-			t.Skip(err.Error())
-			return
-		}
+	if conn, err := sql.Open("proton", "proton://127.0.0.1:8463"); assert.NoError(t, err) {
+		//if err := checkMinServerVersion(conn, 21, 12); err != nil {
```
Comment (qijun-niu-timeplus) [inline tests/std/geo_ring_test.go]: remove

File: tests/std/lowcardinality_test.go [hunk 24]
```diff
@@ -32,22 +32,22 @@ func TestStdLowCardinality(t *testing.T) {
 	ctx := proton.Context(context.Background(), proton.WithSettings(proton.Settings{
 		"allow_suspicious_low_cardinality_types": 1,
 	}))
-	if conn, err := sql.Open("proton", "proton://127.0.0.1:9000"); assert.NoError(t, err) {
-		if err := checkMinServerVersion(conn, 19, 11); err != nil {
-			t.Skip(err.Error())
-			return
-		}
+	if conn, err := sql.Open("proton", "proton://127.0.0.1:8463"); assert.NoError(t, err) {
+		//if err := checkMinServerVersion(conn, 19, 11); err != nil {
```
Comment (qijun-niu-timeplus) [inline tests/std/lowcardinality_test.go]: remove

File: tests/std/map_test.go [hunk 25]
```diff
@@ -25,19 +25,19 @@ import (
 )
 
 func TestStdMap(t *testing.T) {
-	if conn, err := sql.Open("proton", "proton://127.0.0.1:9000"); assert.NoError(t, err) {
-		if err := checkMinServerVersion(conn, 21, 9); err != nil {
-			t.Skip(err.Error())
-			return
-		}
+	if conn, err := sql.Open("proton", "proton://127.0.0.1:8463"); assert.NoError(t, err) {
+		//if err := checkMinServerVersion(conn, 21, 9); err != nil {
```
Comment (qijun-niu-timeplus) [inline tests/std/map_test.go]: remove

File: tests/issues/482_test.go (hunk 10 above)
Comment (leo-cai-timeplus) [reply tests/issues/482_test.go:69]: porting clickhouse
<https://github.com/ClickHouse/clickhouse-go/blob/main/tests/issues/482_test.go#L62>

**Review by qijun-niu-timeplus — APPROVED** [review]

**Pull request merged**